### PR TITLE
Update project.js and task.js

### PR DESCRIPTION
**Pull request by siddy20**

### CODE REVIEW

File: src/models/project.js
Comment (utkarsha30) [inline src/models/project.js:1]: add more status options in tasks

File: src/models/task.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+const tasksSchema = mongoose.Schema({
+  Title: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+    required: true,
+  },
+  startdate: {
```
Comment (utkarsha30) [inline src/models/task.js:14]: duplicate values

File: src/models/project.js
Comment (siddy20) [reply src/models/project.js:1]: krdiya resolve

**Review by utkarsha30 — CHANGES_REQUESTED** [review]
make changes

**Review by utkarsha30 — APPROVED** [review]

**Pull request merged**